omnibus/config/software/haproxy.rb

```diff
@@ -44,27 +44,17 @@
    "USE_PCRE_JIT" => "1",
    "USE_ZLIB" => "1",
    "USE_OPENSSL" => "1",
+   # Required to resolve hostnames to IPv6 addresses
+   # off-by-default because of prolems on older glibc's
+   # TODO(ssd): Should we turn this off on RHEL5?
```

Comment:
If we don't support RHEL5 then this TODO can go as well.

Reply:
I've removed that noww